.circleci/config.yml

```diff
@@ -381,6 +381,11 @@ jobs:
           clone_options: '--depth 1 --verbose'
       - attach_workspace:
           at: .
+      - run:
+          # Although playwright was downloaded as part of the build step, it is faster to re-install
+          # now that save and restore it into the cache multiple times
+          name: Download Playwright
+          command: npx playwright install
```

Comment:
This should not be necessary at all, if you're running in the right container this should already be done!

Reply:
Oh, good point. I'll look into that.